charts/seed-bootstrap/values.yaml

```diff
@@ -77,3 +78,6 @@ fluentd-es:
   enabled: true
   images:
     fluentd-es: image-repository:image-tag
+
+cert-manager:
+  enabled: true
```

Comment:
how about naming this `certificate-management` like in the cert-broker?

Reply:
Agree with @rfranzke and renamed `certificate-management` to `cert-broker` for Shoot core addons to be consistent.